Sources/PubNubChatComponents/Message List/MessageListViewController.swift

```diff
@@ -82,6 +98,63 @@ open class ChatViewController: CollectionViewComponent {
     })
   }
   
+  open override func onLongPressGestureRecognized(gesture: UIGestureRecognizer) {
+    
+    if gesture.state == .began {
+      
+      // Finds the cell that was pressed
+      guard let affectedCell = collectionView.visibleCells.first(where: {
+        $0.frame.contains(gesture.location(in: collectionView))
+      }) as? MessageListItemCell else {
+        return
+      }
+      
+      let reactionList = affectedCell.reactionListView.allReactions.map() { $0.reaction }
+      let pickerView = AddMessageReactionComponent.DefaultPickerView(reactionList: reactionList)
+      
+      // Creates and configures a view controller responsible for displaying the emoji picker view
+      let viewController = AddMessageReactionComponent(pickerView: pickerView, reactionList: reactionList)
+      viewController.modalPresentationStyle = .custom
+      viewController.transitioningDelegate = self
+      
+      // Captures selection from the emoji picker view
+      viewController
+        .selectedReactionValue
+        .sink(receiveValue: { [weak self] value in
+          self?.onMessageReactionSelected(with: value, for: affectedCell)
```

Comment:
affectedCell might need to be made weak here to avoid retain cycles. `[weak self, weak affectedCell]`